src/main/java/com/newrelic/telemetry/opentelemetry/export/NewRelicSpanExporter.java

```diff
@@ -52,19 +50,15 @@
    */
   @Override
   public ResultCode export(List<SpanData> openTelemetrySpans) {
-    try {
-      SpanBatch spanBatch = adapter.adaptToSpanBatch(openTelemetrySpans);
-      spanBatchSender.sendBatch(spanBatch);
-      return ResultCode.SUCCESS;
-    } catch (RetryWithRequestedWaitException | RetryWithBackoffException e) {
-      return ResultCode.FAILED_RETRYABLE;
-    } catch (ResponseException e) {
-      return ResultCode.FAILED_NOT_RETRYABLE;
-    }
+    SpanBatch spanBatch = adapter.adaptToSpanBatch(openTelemetrySpans);
+    telemetryClient.sendBatch(spanBatch);
+    return ResultCode.SUCCESS;
```

Comment:
It feels weird that we always return `SUCCESS` now.  Perhaps the export method should also be made void?

Reply:
yep. That will happen in a future version.